include/Core/Config.php

```diff
@@ -132,15 +132,33 @@ public static function set($family,$key,$value) {
 		$dbvalue = (is_array($value)?serialize($value):$value);
 		$dbvalue = (is_bool($dbvalue) ? intval($dbvalue) : $dbvalue);
 
-		$ret = q("INSERT INTO `config` ( `cat`, `k`, `v` ) VALUES ( '%s', '%s', '%s' )
-ON DUPLICATE KEY UPDATE `v` = '%s'",
+		// The "INSERT" command is very cost intense. It saves performance to do it this way.
```

Comment:
as config is mostly read from in-memory cache after being loaded from db (unless $refresh false) and as every script should end with session closing , calling [ref_session_close() in include/session.php](https://github.com/friendica/friendica/blob/97020817225a8762b699a836da46ba6eed59a162/include/session.php#L59), can't we delay the db write at session closing with a "mass insert" ?